tests/unit/test_additionalfilescache.py

```diff
@@ -0,0 +1,38 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+from PyInstaller.depend.imphook import AdditionalFilesCache
+import pytest
+
+
+@pytest.mark.parametrize(
```

Comment:
Am I reading this correctly that you parameterize this test, but then only provide one set of parameters? If so, why parametrize?

Reply:
Oh I meant to clean that up. I'll get onto this and writing more tests.

Reply:
Cleaned up. @bjones1 another review please :-)